driver/src/main/scala/api/collections/UpdateOps.scala

```diff
@@ -170,22 +170,23 @@ trait UpdateOps[P <: SerializationPack] extends UpdateCommand[P]
      * }
      * }}}
      */
-    final def many(updates: Iterable[UpdateElement])(implicit ec: ExecutionContext): Future[MultiBulkWriteResult] = updates.headOption match {
-      case Some(first) => {
+    final def many(updates: Iterable[UpdateElement])(implicit ec: ExecutionContext): Future[MultiBulkWriteResult] = {
+      if (updates.nonEmpty) {
         val bulkProducer = BulkOps.bulks(
-          updates, maxBsonSize, metadata.maxBulkSize) { up =>
-          elementEnvelopeSize + pack.bsonSize(up.q) + pack.bsonSize(up.u)
+          updates, maxBsonSize, metadata.maxBulkSize) {
```

Comment:
Please do not break format. Revert that.

Reply:
Done.